src/prompt_toolkit/layout/screen.py

```diff
@@ -169,7 +169,7 @@ def __init__(
 
         #: Escape sequences to be injected.
         self.zero_width_escapes: defaultdict[int, defaultdict[int, str]] = defaultdict(
-            lambda: defaultdict(lambda: "")
+            lambda: defaultdict(str)
```

Comment:
I'm not yet going to drop Python 3.7 yet. It's still too much in use today: https://pypistats.org/packages/prompt-toolkit
However, I'll cherry-pick this line, which works fine on 3.7 too.